CHANGELOG/1.2-CHANGELOG.md

```diff
@@ -0,0 +1,424 @@
+# v1.2 Changelog
+
+## Table of Contents
+
+- [v1.2.0-rc1](#v120-rc1)
+
+# v1.2.0-rc1
+
+We expect that this release candidate is quite close to the final v1.2.0
+release. However, subsequent breaking API changes are still possible.
+
+This release candidate is suitable for implementors, but we do not recommend
+shipping products based on a release candidate API due to the possibility of
+incompatible changes prior to the final release. The following represents the
+changes since v1.1.0:
+
+## Release Cycle changes
+
+In this release, we moved to using a phased release cycle modeled on the
+[upstream Kubernetes release cycle](https://kubernetes.io/releases/release/).
+
+There are now four phases:
+
+* **Scoping**: 4-6 weeks, where the maintainers and community determine the
+  set of features we want to include in the release. A key deliverable here
+  is ensuring that the Experimental channel does not get any bigger than it
+  already is; we want to ensure that features make their way to Standard or
+  are removed.
+* **GEP Iteration and Review**: 5-7 weeks, where we work to update GEPs, and
+  solidify and meet graduation criteria for in-scope GEPs.
+* **API Refinement and Documentation**: 3-5 weeks, where we translate proposed
+  changes from GEPs into changes to both the API specification and
+  documentation.
+* **SIG-Network Review and Release Candidates**: 2-4 weeks, where we perform
+  our required upstream reviews, get any further required changes in, and
+  release our release candidates so that implementations can get started
+  with work on the new features asap.
+
+For all the detail about this, please see the [Release Cycle docs](https://gateway-api.sigs.k8s.io/contributing/release-cycle/).
+
+Relevant PRs:
+* Adding Release Cycle Documentation by @robscott in [#3063](https://github.com/kubernetes-sigs/gateway-api/pull/3063)
+
+## Standard Channel Additions and Changes
+
+The Standard channel is Gateway API's set of maximally-stable install files.
+Only features with the best testing and support are added to the standard
+channel. This channel should be considered GA or stable, and future changes
+will only be compatible changes.
+
+### Infrastructure Labels and Annotations moved to Standard 🎉
```

Comment:
Non-blocking nit, feel free to ignore: Since this is already in a "Graduating to standard" section, we probably don't need that repeated in each subheading.
```suggestion
### Infrastructure Labels and Annotations
```